### PR TITLE
Collect more info from namespaces

### PR DESCRIPTION
**Pull request by subyssurendran666**
Collect more informations from ibm-common-services namespace rather than csv, InstallPlans and subscriptions

### CODE REVIEW

File: generate_postmortem.sh [hunk 1]
```diff
@@ -1457,83 +1462,83 @@ for NAMESPACE in $NAMESPACE_LIST; do
             fi
         done
     fi
-    #^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-done
```
Comment (subindani) [inline generate_postmortem.sh:1461]: @subyssurendran666 Could you please explain why is this loop removed here and added in Line 1539 ?

**Review by subindani — CHANGES_REQUESTED** [review]
@subyssurendran666 Thank you adding new functionality to the tool. Could you please let this on :
  a) k8s APIC deployment
  b) cp4i APIC deployment 
  c) Openshift APIC deployment
and share the logs.zip file at box.com or location in fyre with me ? 

Ideally, we don't want the tool to create ibm-common-services directories for deployment where this namespace does not exist. I just tested this now on my API deployment on k8s and noticed it :
```
kubernetes/namespaces/ibm-common-services$ find .
.
./subscriptions
./subscriptions/describe
./subscriptions/yaml
./cluster_service_version
./cluster_service_version/describe
./cluster_service_version/yaml
```

File: generate_postmortem.sh (hunk 1 above)
Comment (subyssurendran666) [reply generate_postmortem.sh:1461]: @subindani The requirement is to collect all the normal data for any namespace in addition to cluster service version, install plans and subscriptions. That's why I have moved the codes to find the csv, ip and subscription information inside the for loop.

Comment (subindani) [reply generate_postmortem.sh:1461]: Thank you @subyssurendran666

**Review by subindani — APPROVED** [review]

**Comment by subyssurendran666** [comment]
@subindani Sorry for the late reply. I've tested the postmortem script in K8s, OVA, CP4i and OCP platforms. You could find the logs [here](https://ibm.box.com/s/kgntzm0c619t1h1ir9i72phxwexhsftj).

**Comment by subindani** [comment]
@subyssurendran666 I will reach out to you in slack

**Pull request merged**